### ISSUE TITLE
[Bug]: ControlNet v1.1.410 Preprocessor Preview fail to execute and the resulting image using Generate button is not as espected, it look ultra zoomed portion of the image loaded

### ISSUE THREAD

**Issue by Poukpalaova** [Mikubill/sd-webui-controlnet#2149]
### Is there an existing issue for this?

- [X] I have searched the existing issues and checked the recent builds/commits of both this extension and the webui

### What happened?

Trying to use controlNet v1.1.410 just doesn't work. Using the preprocessor create a black image in the preprocessor preview. This happen in txt2img and img2img.

### Steps to reproduce the problem

1. Open SD 1.6.0
2. Load a model, any model and the wanted VAE
3. Put normal generation setting with a simple prompt with an image 512x512
4. Use that generated image in the controlNet single image
5. Enable controlNet
6. Use a control type, like Canny
7. Activate the Allow Preview box
8. Activate Pixel Perfect
9. Click the small red explosion button between preprocessor and model
10. This loading a black picture in the Preprocessor Preview

### What should have happened?

It should have load a black picture with the white lines of the loaded image.
![2023-09-27_14-16-35](https://github.com/Mikubill/sd-webui-controlnet/assets/141651008/9134dc9a-6497-4826-9230-c5922602d231)

### Commit where the problem happens

version: [v1.6.0] Commit hash: 5ef669de080814067961f28357256e8fe27544f4
python: 3.10.11
torch: 2.0.1+cu118
xformers: 0.0.22+1e065bc.d20230927
gradio: 3.41.2
checkpoint: photon_v1.safetensors
controlNet: v1.1.410

### What browsers do you use to access the UI ?

Google Chrome, Microsoft Edge

### Command Line Arguments

```Shell
COMMANDLINE_ARGS= --xformers --opt-sub-quad-attention --opt-channelslast
```

### List of enabled extensions

a1111-sd-webui-tagcomplete
adetailer
miaoshouai-assistant
model-keyword
sd-dynamic-prompts
sd-extension-system-info
sd-model-organizer
sd-webui-agent-scheduler
sd-webui-aspect-ratio-helper
sd-webui-controlnet
sd-webui-depth-lib
sd-webui-infinite-image-browsing
sd-webui-inpaint-anything
sd-webui-openpose-editor
sd-webui-reactor
sd-webui-segment-anything
stable-diffusion-webui-rembg
stable-diffusion-webui-wildcards
ultimate-upscale-for-automatic1111

### Console logs

```Shell
2023-09-27 14:16:06,832 - ControlNet - STATUS - Preview Resolution = 1024
2023-09-27 14:23:32,606 - ControlNet - STATUS - Loading model: control_v11p_sd15_canny [d14c016b]
2023-09-27 14:23:42,817 - ControlNet - STATUS - Loaded state_dict from [F:\IMAGES\AIgenerated\SDXL 1.6\stable-diffusion-webui\extensions\sd-webui-controlnet\models\control_v11p_sd15_canny.pth]
2023-09-27 14:23:42,818 - ControlNet - STATUS - controlnet_default_config
2023-09-27 14:23:45,110 - ControlNet - STATUS - ControlNet model control_v11p_sd15_canny [d14c016b] loaded.
2023-09-27 14:23:45,188 - ControlNet - STATUS - using mask as input
2023-09-27 14:23:45,191 - ControlNet - STATUS - Loading preprocessor: none
2023-09-27 14:23:45,191 - ControlNet - STATUS - preprocessor resolution = 1024
2023-09-27 14:23:45,457 - ControlNet - STATUS - ControlNet Hooked - Time = 14.009279012680054
100%|██████████████████████████████████████████████████████████████████████████████████| 20/20 [00:04<00:00,  4.33it/s]
Miaoshouai boot assistant: Memory Released!████████████████████████████████████████████| 20/20 [00:03<00:00,  5.45it/s]
Total progress: 100%|██████████████████████████████████████████████████████████████████| 20/20 [00:05<00:00,  3.75it/s]
Total progress: 100%|██████████████████████████████████████████████████████████████████| 20/20 [00:05<00:00,  5.45it/s]
```

### Additional information

The result is nothing like it should be and the consol give no error BUT for this line that look strange to me:
2023-09-27 14:23:45,191 - ControlNet - STATUS - Loading preprocessor: none
As I choosed preprocessor canny.
It look like nothing is loading in preprocessor, should it be?
![2023-09-27_14-25-48](https://github.com/Mikubill/sd-webui-controlnet/assets/141651008/dbd15433-7347-48ee-a368-db88e8461841)

**Comment by huchenlei** [comment]
See https://github.com/Mikubill/sd-webui-controlnet/issues/1651

**Comment by huchenlei** [comment]
You can check following option in settings to prevent
```
2023-09-27 14:23:45,188 - ControlNet - STATUS - using mask as input
```

![Screen Capture 030 - Stable Diffusion - localhost](https://github.com/Mikubill/sd-webui-controlnet/assets/20929282/c03ec236-d3bc-4556-914a-ca482f088f44)

**Comment by Poukpalaova** [comment]
> You can check following option in settings to prevent
> 
> ```
> 2023-09-27 14:23:45,188 - ControlNet - STATUS - using mask as input
> ```
> 
> ![Screen Capture 030 - Stable Diffusion - localhost](https://user-images.githubusercontent.com/20929282/271082041-c03ec236-d3bc-4556-914a-ca482f088f44.jpg)

Wow, I've been looking for week to resolve that, tryed 3 diff SD install, talked on forum. Why this option is not enabled by default? If it is not used as default, why other person ignor about this problem and have no issue using control net?

**Comment by huchenlei** [comment]
I think the original intention of the mask in non-inpaint control types is to serve as a quick way of scribble something. However, according to https://github.com/Mikubill/sd-webui-controlnet/issues/1651, gradio has the mysterious issue that sometimes even the user has not scribbled anything on the canvas but the mask we got has some random 1s in it. We have increased the threshold from 0 to 5 to counter those random 1s, this resolved problem for most people.

**Comment by Su13n** [comment]
> I think the original intention of the mask in non-inpaint control types is to serve as a quick way of scribble something. However, according to #1651, gradio has the mysterious issue that sometimes even the user has not scribbled anything on the canvas but the mask we got has some random 1s in it. We have increased the threshold from 0 to 5 to counter those random 1s. This has resolved problem for most people.

What can I do if enabling the setting still does not hide masks?
